o/opentelemetry-cpp/opentelemetry-cpp_ubi_8.5.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package	: opentelemetry-cpp
+# Version	: v1.4.1 
+# Source repo	: https://github.com/open-telemetry/opentelemetry-cpp
+# Tested on	: ubi 8.5
+# Language      : node
+# Travis-Check  : False
```

Comment:
why is travis check false?

Reply:
because the script takes more than 50M and the travis check times out.

Reply:
ok, let's keep it disabled then